awx/plugins/inventory/foreman.py

```diff
@@ -46,6 +46,8 @@
 
 from requests.auth import HTTPBasicAuth
 
+from ansible.module_utils._text import to_text
```

Comment:
https://github.com/ansible/ansible/blob/stable-2.4/lib/ansible/module_utils/_text.py#L169

this should be compatible with Ansible 2.3 and onwards.